ecosystem/sep-0009.md

```diff
@@ -30,39 +37,66 @@ occupation | 3 | [ISCO08](https://en.wikipedia.org/wiki/International_Standard_C
+'KYC2' | `postal_code` | string | Postal or other code identifying user's locale
+'KYC2' | `address` | string | Entire address (country, state, postal code, street address, etc...) as a multi-line string
+'KYC0' | `mobile_number` | phone number | Mobile phone number with country code, in E.164 format
+'KYC0' | `email_address` | string | Email address
+'KYC1' | `birth_date` | date | Date of birth, e.g. `1976-07-04`
+'KYC2' | `birth_place` | date | Place of birth (city, state, country; as on passport)
+'KYC2' | `birth_country_code` | country code | ISO Code of country of birth
+'KYC2' | `bank_account_number` | string | Number identifying bank account
+'KYC3' | `bank_number` | string | Number identifying bank in national banking system (routing number in US)
+'KYC3' | `bank_phone_number` | string | Phone number with country code for bank
+'KYC3' | `tax_id` | string | Tax identifier of user in their country (social security number in US)
+'KYC3' | `tax_id_name` | string | Name of the tax ID (`SSN` or `ITIN` in the US)
+'KYC3' | `occupation` | number | Occupation ISCO code
+'KYC3' | `employer_name` | string | Name of employer
+'KYC3' | `employer_address` | string | Address of employer
+'KYC2' | `language_code` | language code | primary language
+'KYC1' | `id_type` | string | `passport`, `drivers_license`, `id_card`, etc...
+'KYC1' | `id_country_code` | country code | country issuing passport or photo ID as ISO 3166-1 alpha-3 code
+'KYC1' | `id_issue_date` | date | ID issue date
+'KYC1' | `id_expiration_date` | date | ID expiration date
+'KYC1' | `id_number` | string | Passport or ID number
+'KYC1' | `photo_id_front` | binary | Image of front of user's photo ID or passport
+'KYC1' | `photo_id_back` | binary | Image of back of user's photo ID or passport
+'KYC3' | `notary_approval_of_photo_id` | binary | Image of notary's approval of photo ID or passport
+'KYC3' | `ip_address` | string | IP address of customer's computer
+'KYC2' | 'photo_proof_residence' | binary | Image of a utility bill, bank statement with the user's name and address
+
+Legal entity KYC fields
+
+KYC Level | Name | Type | Description
+-----------|-----|------|------------
+'KYC0' | `company_name' | string | Full company name as on the company incorporation papers
+'KYC0' | `company_VAT_number' | string | Company VAT number 
+'KYC0' | `company_registration_number' | string | Company registration number
+'KYC0' | `company_registered_address' | string | Company registered address
+'KYC1' | `number_of_shareholders' | number | Company shareholder number
+'KYC1' | `shareholder_name' | string | Can be a company or a person and should be queried recursively up to the ultimate beneficial owners (with KYC information for natural persons such as above)
+'KYC0' | `image_incorporation_doc' | string | Image of company incorporation documents
+'KYC1' | `image_proof_adress' | string | Image of a utility bill, bank statement with the company's name and address
```

Comment:
Since this is the legal entity analog of `photo_proof_residence`, I'd try to make their names more consistent.

Also, since we already have a pattern of using `photo` instead of `image`, I'd stick to `photo` in these fields.